source/main/commands/file-path-find.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @ignore
+ * BEGIN HEADER
+ *
+ * Contains:        FilePathFind command
+ * CVM-Role:        <none>
+ * Maintainer:      Hendrik Erz
+ * License:         GNU GPL v3
+ *
+ * Description:     This command finds the absolute path of a file
+ *
+ * END HEADER
+ */
+
+import ZettlrCommand from './zettlr-command'
+import { filetypes as FILETYPES } from '../../common/data.json'
+
+export default class FilePathFind extends ZettlrCommand {
+  constructor (app: any) {
+    super(app, ['file-path-find'])
+  }
+
+  /**
+      * Removes a file.
+      * @param {String} evt The event name
+      * @param  {Object} arg the parameters of the file to be deleted
+      * @return {Boolean} Whether the file was successfully deleted.
+      */
+  async run (evt: string, arg: any): Promise<String> {
+    // Initialise the file as the result of findExact failing
+    let file
+    // It might be linked by ID
+    file = this._app.getFileSystem().findExact((arg as string), 'id')
+    if (file !== undefined) {
+      return file.path
+    }
+    // It's not an ID, so search each type of file
+    for (let type of FILETYPES) {
+      file = this._app.getFileSystem().findExact((arg as string) + type, 'name')
+      if (file !== undefined) {
+        // If we find it, then return it
+        return file.path
+      }
+    }
+    // We can't find it, so return Not Found
+    return 'Not Found'
+  }
+}
+
+module.exports = FilePathFind
```

Comment:
Remove this line, that's the old NodeJS module system, and you already have `export default class` above, as it's supposed to be.

Plus, I think it might sound nicer if you had it named `FileFindPath` (or, with my changes, a different, more fitting name) – don't feel _too_ obliged to copy my "But File/Dir has to go first!!"-approach if that makes it sound like complete nonsense …